packages/app/client/src/components/projects/overview/components/ProjectOverviewForm.js

```diff
@@ -53,26 +66,30 @@ function ProjectOverviewForm(props) {
             ...props
           }) => (
             <form noValidate onSubmit={props.handleSubmit}>
-              <Typography
-                className={classes.header}
-                variant="subtitle1"
-              >
+              <Typography className={classes.header} variant="subtitle1">
                 Project Photo
               </Typography>
-              <div className={classes.image}>
-                <img src={values.attachment} alt="LoadProjectImage" />
-              </div>
-              <Typography
-                className={classes.header}
-                variant="subtitle1"
-              >
+              {values.attachment ? (
+                <img
+                  className={classes.image}
+                  src={values.attachment}
+                  alt="LoadProjectImage"
+                />
+              ) : (
+                <img
+                  className={classes.image}
+                  src={ProjectPicture}
+                  alt="LoadProjectImage"
```

Comment:
The only thing that changes is the src property. We could move the logic to avoid repetition.
`src={values.attachment || ProjectPicture}`

Reply:
okey, I'll do it

Reply:
did it